### ISSUE TITLE
IonButton onClick event called twice with React @ionic/react

### ISSUE THREAD

**Issue by josepol** [ionic-team/ionic#17292]
# Bug Report

**Ionic version:**

[x] **4.x**

**Current behavior:**
IonButton component onClick event is being called twice with Ionic 4 with React

**Expected behavior:**
IonButton component onClick event should be called once

**Steps to reproduce:**
Create a new proyect with create-react-app
Install @ionic/react and @ionic/core as it is explained in https://www.npmjs.com/package/@ionic/react
Call registerIonic() on index.js
Import IonButton with import { IonButton } from '@ionic/react';
Declare IonButton component and add it the onClick event

Test it with console.log()

```
class App extends Component {

  incrementCount(e) {
    console.log('+++');
}

  decrementCount(e) {
    console.log('---');
  }

  render() {
    return (
      <div className="App">
        {/* This is called twice */}
        <IonButton onClick={(e) => this.incrementCount(e)}>Ionic Button +</IonButton>
        <IonButton onClick={(e) => this.decrementCount(e)}>Ionic Button -</IonButton>
        {/* This is called once */}
        <button  onClick={(e) => this.incrementCount(e)}>+</button>
        <button  onClick={(e) => this.decrementCount(e)}>-</button>
      </div>
    );
  }
}

export default App;
```

Am I doing something wrong? I have tested it in Chrome and Firefox

Thank you very much for your support!!

**Comment by silentHoo** [comment]
I've exactly found the same problem with my code and wanted to open an issue, too. The code to reproduce would be the same.

**Comment by manucorporat** [comment]
cc @jthoms1

**Comment by jthoms1** [comment]
This fixed in release 0.0.4 of @ionic/react
